code/modules/augment/passive/faulty/noisemaker.dm

```diff
@@ -0,0 +1,6 @@
+/obj/item/organ/internal/augment/faulty/noisemaker
+	name = "noisemaker"
+	desc = "An augment that was badly installed. Sometimes it emits a loud noise."
+
+/obj/item/organ/internal/augment/faulty/noisemaker/on_malfunction()
+	owner.audible_message(SPAN_DANGER("[owner]'s body emits a loud, piezoelectric beep."), hearing_distance = 7)
```

Comment:
You should actually play a beep noise for players

Reply:
I can't find a SFX for an electric screech/hiss/beep/whine/awful noise.

Reply:
I refuse to believe there's not a suitable one in the `sound/machines/` folder

Reply:
Not one that's close enough to my mental imagination of the sound.

Reply:
https://freesound.org/people/waveplay./sounds/515521/ I found something a little closer.

Reply:
https://freesound.org/people/waveplay./sounds/395542/ Maybe?

Reply:
this is very, very close https://freesound.org/people/waveplay./sounds/389400/

Reply:
Okay, that sort of sound did not really come across in the description at all

Reply:
I know! It's so hard to write good words.

Reply:
Could try the airlock-crowbar noise sped up (frequency var, but it also pitch-shifts it), possibly reversed. That'd prolly sound sufficiently grating